app/_src/kubernetes-ingress-controller/concepts/architecture.md

```diff
@@ -14,7 +14,17 @@ The {{site.kic_product_name}} configures {{site.base_gateway}} using Ingress or
 
 The figure illustrates how {{site.kic_product_name}} works:
 
-![high-level-design](/assets/images/products/kubernetes-ingress-controller/high-level-design.png "High Level Design")
+<!--vale off-->
+{% mermaid %}
+flowchart TD
+    subgraph Kubernetes cluster
+        direction LR
+        A(<div style="text-align:center;"><img src="/assets/images/icons/third-party/kubernetes-logo.png" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/> API server) --> |events| B(<div style="text-align:center;"><img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Controller)
+        B --> |configuration| C(<div style="text-align:center;"><img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Kong)
```

Comment:
```suggestion
        A(<img src="/assets/images/icons/third-party/kubernetes-logo.png" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/> API server) --> |events| B(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Controller)
        B --> |configuration| C(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Kong)
```
Try it without these "div style" tags. They're not even closed, so not sure why they're working on mermaid.live